backend/schemas/project.json

```diff
@@ -4,12 +4,12 @@
     "properties": {
         "callUrl": {
             "format": "uri",
-            "pattern": "^https://doi.org/.*$",
+            "pattern": "^https?://doi.org/.*$",
```

Comment:
https here was deliberate, it helps with search engine ranking to not have http links

Reply:
I re-instated the requirement for secure urls except for project home urls. Some projects only have a standard home url, like ABC-MUSE: http://www.amusecode.org/